toolkit/tools/internal/safechroot/safechroot.go

```diff
@@ -689,3 +694,33 @@ func (c *Chroot) GetMountPoints() []*MountPoint {
 	mountPoints := append([]*MountPoint(nil), c.mountPoints...)
 	return mountPoints
 }
+
+// stopGPGAgent stops gpg-agent and keyboxd if they are running inside the chroot.
+//
+// It is possible that one of the packages or post-install scripts started a GPG agent.
```

Comment:
I'd even simplify and say that a GPG agent might have been started while the chroot was in use. There's many ways we use chroots, no need to suggest it's only a package installation, which might trigger the agent.

Reply:
Updated!